project/src/main/webapp/sps/data/Event.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Date;
+import java.util.Random; 
+import java.util.Set;
+
+/** Class containing Event object 
+Setters for variables that user can change about event
+*/
+public class Event {
+  private final int eventId;
+  private String name;
+  private String description;
+  private Set<String> labels;
+  private String location;
+  private Date date;
+  private Set<VolunteeringOpportunity> opportunities;
+  private Set<User> attendees;
+  private User host;
+
+  Random rand = new Random(); 
+
+  /** TO DO (MVP): Add Event to Event db*/
+  public Event(String name, String description, Set<String> labels, String location, Date date, 
```

Comment:
To me it makes more sense to just initialize attendees as empty in the constructor, and then add attendees as people sign up

Reply:
thanks done!